pubspec.yaml

```diff
@@ -39,7 +39,7 @@ flutter:
     platforms:
       android:
         package: io.customer.customer_io
```

Comment:
I think we can rename package to follow more conventional format, but I avoided doing it for now. Happy to make the change if we feel we should go for it.